packages/viewer/src/scenes/operations.ts

```diff
@@ -16,12 +20,14 @@ export interface ChangeMaterialOperation {
 export type ItemOperation =
   | ShowItemOperation
   | HideItemOperation
-  | ChangeMaterialOperation;
+  | ChangeMaterialOperation
+  | ClearItemOperation;
 
 export interface SceneItemOperations<T> {
   materialOverride(color: ColorMaterial): T;
   show(): T;
   hide(): T;
+  clear(): T;
```

Comment:
Do we want to call this `.clear()` or should we call this `.clearOverrides()`?

Not sure what sort of verbiage we want to expose in the sdk